akka-projection-cluster/src/main/scala/akka/projection/cluster/internal/ClusterProjectionRunnerImpl.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.cluster.internal
+
+import java.util.function.Supplier
+
+import scala.collection.immutable
+import scala.jdk.CollectionConverters._
+
+import akka.actor.typed.ActorSystem
+import akka.annotation.InternalApi
+import akka.cluster.sharding.typed
+import akka.cluster.sharding.typed.ShardedDaemonProcessSettings
+import akka.projection.Projection
+import akka.projection.internal.ProjectionBehavior
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[akka] object ClusterProjectionRunnerImpl {
+
+  /**
+   * Scala API
+   */
+  def init[Envelope](
+      system: ActorSystem[_],
+      projectionName: String,
+      projections: immutable.IndexedSeq[() => Projection[Envelope]],
+      shardedDaemonSettings: ShardedDaemonProcessSettings): Unit = {
+
+    typed.scaladsl
+      .ShardedDaemonProcess(system)
+      .init[ProjectionBehavior.Command](
+        s"cluster-projection-runner-$projectionName",
+        projections.size - 1,
```

Comment:
The sharded daemon does:

```scala
val entityIds = (0 to numberOfInstances).map(_.toString)
```

If size is 2, we will have 0, 1 and 2. 

This is actually a bug to fix there, because it actually creating one extra.

btw, here I was getting the out-of-bound I mentioned

Reply:
that was indeed a bug, but I think it's fixed in 2.6.5 (at least it is correct in master)

Reply:
it's fixed in 2.6.5 and it's merged here already. Will remove the hack.